ivy/functional/ivy/nest.py

```diff
@@ -729,7 +729,7 @@ def copy_nest(
     nest: Union[ivy.Array, ivy.NativeArray, Iterable],
     include_derived: bool = False,
     to_mutable: bool = False,
-) -> Union[ivy.Array, ivy.NativeArray, Iterable, Dict]:
+) -> Union[ivy.Array, ivy.NativeArray, Iterable]:
```

Comment:
Please revert all changes made to this file as it's not related to the function you're working on.